code/game/objects/effects/temporary_visuals/miscellaneous.dm

```diff
@@ -141,6 +141,23 @@
 	. = ..()
 	update_icon()
 
+/obj/effect/temp_visual/bsa_splash
+	name = "Bluespace Energy Waves"
```

Comment:
Should probably not be capitalized. If name ends with "s", should also set `gender = PLURAL` for examine messages to render properly.